doc/man7/migration_guide.pod

```diff
@@ -101,6 +101,11 @@ should ensure that the legacy provider has been loaded. This can be achieved
 either programmatically or via configuration. See L<crypto(7)> man page for
 more information about providers.
 
+If the application code uses any of such algorithms via the EVP interface, the
+application should verify the result of the L<EVP_EncryptInit(3)>,
+L<EVP_EncryptInit_ex(3)>, and L<EVP_DigestInit(3)> functions. In case when
+the requested legacy algorithm is not available, these functions will fail.
```

Comment:
IMO this is giving false impression that non-legacy algorithms cannot fail. Applications should always check the return codes if they want to work correctly. So please adjust the text somewhat.

Reply:
Fixed